.github/workflows/instance_types.yaml

```diff
@@ -0,0 +1,28 @@
+name: Update Instance Types
+
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: '0 0 * * 0'
+
+jobs:
+  update-instance-types:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up system
+      run: bin/before_install
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.7
+        bundler-cache: true
+    - name: Run instance_types rake task
+      run: bundle exec rake aws:extract:instance_types
+    - name: Create Pull Request
+      uses: peter-evans/create-pull-request@v4
+      with:
+        add-paths: db/fixtures/aws_instance_types.yml
+        commit-message: Update AWS instance_types
+        title: Update AWS instance_types
```

Comment:
Maybe for a follow-up , but I'm think we should set `body:` here as well.  The default one isn't very informative.

Reply:
I'd be fine with leaving blank as the title is informative enough, or perhaps we could add in the date?